### DISCUSSION TITLE
I am having trouble with computer engineering and I wanted to know any websites I could use to help

### DISCUSSION THREAD

**Discussion by ghost** [community/community#67126] [Programming Help]
### Body

I am taking Computer Engineering grade 11 but forgot most of last year and when relearning it none of it sticks in. Are there any resources someone with an IQ <70 could use?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ghost** [comment]
PLEASE HELP I eventually have a test on binary and hexadecimal conversions

**Comment by elendil7** [comment]
### Introduction

When I was studying Foundations of Computing at University (maths-based module), I found the following resources to be highly helpful, which explain the concepts in depth. It does admittedly involve a bit of memorization, but the videos below show the easiest and most efficient way of performing the calculations. There are also various tools you can use for converting hex-octal, binary-decimal, and so on. I'll link these below.

### Youtube videos (mainly from the Organic Chemistry Tutor):

- [ Converting Binary to Hexadecimal ](https://www.youtube.com/watch?v=G5Rpxhtmo3I)
- [ How To Convert Binary to Hexadecimal - Computer Science ](https://www.youtube.com/watch?v=tSLKOKGQq0Y)
- [ Number Systems Introduction - Decimal, Binary, Octal & Hexadecimal ](https://www.youtube.com/watch?v=FFDMzbrEXaE)
- [ How To Convert Hexadecimal to Binary ](https://www.youtube.com/watch?v=D_YC6DSPpQE)
- [ How To Convert Decimal to Hexadecimal ](https://www.youtube.com/watch?v=QJW6qnfhC70)
- [ How To Convert Hexadecimal to Decimal ](https://www.youtube.com/watch?v=pg-HEGBpCQk)
- [ How To Convert Decimal to Binary ](https://www.youtube.com/watch?v=rsxT4FfRBaM)
- [ Binary to Octal Conversion ](https://www.youtube.com/watch?v=JxmarqiqUdM)
- [ Hexadecimal to Octal Conversion ](https://www.youtube.com/watch?v=ML_AkW97Axk)

### Tools
- [Binary to Hex converter](https://www.rapidtables.com/convert/number/binary-to-hex.html)
- [Hex to Binary converer](https://www.rapidtables.com/convert/number/hex-to-binary.html)
